source/stylesheets/components/typography.css

```
@@ -28,6 +28,12 @@
 }
 
 /* paragraphs */
+.par-2xl {
+  `@apply` text-[40px] leading-[48px] md:text-[64px] md:leading-[72px] font-bold;
```

Stylelint (17.10.0)

[error] Unknown word text- (CssSyntaxError)

(CssSyntaxError)